src/servicebus/azext_servicebus/_help.py

```diff
@@ -0,0 +1,476 @@
+"""
+
+helps['servicebus namespace authorizationrule delete'] = """
+    type: command
+    short-summary: Deletes the AuthorizationRule of the Service Bus Namespace.
+    examples:
+        - name: Deletes the AuthorizationRule of the Service Bus Namespace.
+          text: az servicebus namespace authorizationrule delete --resource-group myresourcegroup --namespace-name mynamespace
+           --name myauthorule
+"""
+
+helps['sb queue create'] = """
+    type: command
+    short-summary: Creates the Service Bus Queue
+    examples:
+        - name: Creates Service Bus Queue.
+          text: az servicebus queue create --resource-group myresourcegroup --namespace-name mynamespace --name myqueue
+"""
+
+helps['servicebus queue show'] = """
+    type: command
+    short-summary: shows the Service Bus Queue Details
+    examples:
+        - name: Shows the Service Bus Queue Details
+          text: az servicebus queue show --resource-group myresourcegroup --namespace-name mynamespace --name myqueue
+"""
+
+helps['servicebus queue list'] = """
+    type: command
+    short-summary: List the Queue by Service Bus Namepsace
+    examples:
+        - name: Get the Queues by Service Bus Namespace.
+          text: az servicebus queue list --resource-group myresourcegroup --namespace-name mynamespace
+"""
+
+helps['servicebus queue delete'] = """
+    type: command
+    short-summary: Deletes the Service Bus Queue
+    examples:
+        - name: Deletes the queue
+          text: az servicebus queue delete --resource-group myresourcegroup --namespace-name mynamespace --name myqueue
+"""
+
+helps['servicebus queue authorizationrule create'] = """
+    type: command
+    short-summary: Creates Authorization rule for the given Service Bus Queue
+    examples:
+        - name: Creates Authorization rules for Queue
+          text: az servicebus queue authorizationrule create --resource-group myresourcegroup --namespace-name mynamespace
+           --queue-name myqueue --name myauthorule --access-rights Listen
+"""
+
+helps['servicebus queue authorizationrule show'] = """
+    type: command
+    short-summary: shows the details of AuthorizationRule
+    examples:
+        - name: shows the details of AuthorizationRule
+          text: az servicebus queue authorizationrule show --resource-group myresourcegroup --namespace-name mynamespace
+           --queue-name myqueue --name myauthorule
+"""
+
+helps['servicebus queue authorizationrule list'] = """
+    type: command
+    short-summary: shows the list of AuthorizationRule by Queue
+    examples:
+        - name: shows the list of AuthorizationRule by Queue
+          text: az servicebus queue authorizationrule show --resource-group myresourcegroup --namespace-name mynamespace
+           --queue-name myqueue
+"""
+
+helps['servicebus queue authorizationrule keys list'] = """
+    type: command
+    short-summary: Shows the connectionstrings of AuthorizationRule for the Queue.
```

Comment:
Shouldn't 'connectionstrings' be two words?

Reply:
Resolved